app/src/bootstrap/Kernel.ts

```diff
@@ -60,11 +61,19 @@ export default class Kernel implements KernelInterface {
         try {
             // @See config/app/.ts
             Kernel.logger.info('==== Booting services ====');
-            await Promise.all(
-                this.services.map(
-                    (service) => service.boot?.apply(service, [this.container]),
-                ),
-            );
+
+            const bootGroups = groupBy(this.services, 'bootPriority');
```

Comment:
So this probes every `bootPriority` property in each service?

Reply:
Jup